docs/getting-started/my-account/README.md

```diff
@@ -185,7 +174,7 @@ Also, you can upload a **Custom Avatar** by choosing a Avatar to be uploaded fro
 Note: The optimum size to upload a new profile picture is 128 by 128 pixel. Larger files will be cropped.
 </div>
 
-![Avatar](1571844887353.png)
+![change-avatar](Bildschirmfoto 2022-03-02 um 16.11.12.png)
```

Comment:
please rename the image file and change the name here, so there are no spaces in it.